src/examples/DownloadPdf/DownloadPdf.code.js

```diff
@@ -0,0 +1,25 @@
+import { indent } from "../../utils";
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { DownloadPdf } from "!!./DownloadPdf";
+
+export const downloadPdfCode = `
+<head>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
in our near future, please see about ensuring we don't require jquery for non jquery websites.  i'll leave it be for now.

Reply:
sould be pretty easy to clean up, i'll have a look when later when free